tools/cldr-apps/src/main/webapp/css/redesign.css

```diff
@@ -955,7 +955,7 @@ fieldset[disabled] .btn-primary.active {
 	position:fixed;
 	right:0px;
 	bottom:0px;
-	z-index: 100000000000000000000000000;
+	z-index:800;
```

Comment:
again, just a little nervous about the z-index; do we have more to go on that just that it seems to work OK?

Reply:
I did a search, that's how I found these.

Reply:
fair enough :-)